features/recipes/exceptions.py

```diff
@@ -23,3 +23,15 @@ class InstructionNameViolationException(Exception):
 
 class RecipeWithInstructionNotFoundException(Exception):
     ...
+
+
+class InvalidSortDirection(Exception):
```

Comment:
It is better to do that validation in the input model and decide if you want to continue